api/db/repositories/verification/verification.js

```diff
@@ -0,0 +1,24 @@
+const Customers = require('@db/dbHelper').Customers;
+const { IdNotFound } = require('@helpers/errors');
+
+class Verification {
+    static async DataChecker (checkedInfo, requestedId) {
```

Comment:
the name of this method is unclear for 2 reasons: 
- it's a noun instead of a verb - all functions should be verbs to describe their behaviors which leads me to the second reason: 
- it doesn't explain what it's doing. DataChecker can mean anything and doesn't give you a hint of what you can expect it to do.

Reply:
also the argument name 'checkedInfo' could be changed to 'resource' - that just makes more sense